x-pack/plugins/ml/server/models/file_data_visualizer/import_data.ts

```diff
@@ -89,12 +89,12 @@ export function importDataProvider({ asCurrentUser }: IScopedClusterClient) {
   }
 
   async function createIndex(index: string, settings: Settings, mappings: Mappings) {
-    const body: { mappings: Mappings; settings?: Settings } = {
+    const body: { mappings: Mappings } = {
```

Comment:
`settings?: Settings` is still needed here as an optional property as it is conditionally added bellow on line 102.